xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.cpp

```diff
@@ -170,8 +170,7 @@ bool CActiveAESink::SupportsFormat(const std::string &device, AEAudioFormat &for
           }
           else
           {
-            // TODO - check if sink supports stream type (non IEC packed)
-            return false;
+            return !itt2->m_wantsIECPassthrough;
```

Comment:
As said needs more glue. If a sink can do passthrough is currently
depending on the formats only.

So the new flag only makes sense in this combination. A tristate or sth
would be better.
Am 19.11.2015 9:20 vorm. schrieb "Rainer Hochecker" <
notifications@github.com>:

> In xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.cpp
> <https://github.com/FernetMenta/xbmc/pull/321#discussion_r45312206>:
>
> > @@ -170,8 +170,7 @@ bool CActiveAESink::SupportsFormat(const std::string &device, AEAudioFormat &for
> >            }
> >            else
> >            {
> > -            // TODO - check if sink supports stream type (non IEC packed)
> > -            return false;
> > +            return !itt2->m_wantsIECPassthrough;
>
> this is btw not correct. need to check for supported stream types
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FernetMenta/xbmc/pull/321/files#r45312206>.
>